core/metacling/src/TCling.cxx

```diff
@@ -2026,13 +2035,12 @@ void TCling::RegisterModule(const char* modulename,
       // specifying the relevant include paths we should try loading the
       // modulemap next to the library location.
       clang::Preprocessor &PP = TheSema.getPreprocessor();
-      // Can be nullptr in case of libCore.
-      if (dyLibName)
-         RegisterPrebuiltModulePath(PP, llvm::sys::path::parent_path(dyLibName));
+      RegisterPrebuiltModulePath(llvm::sys::path::parent_path(dyLibName),
+                                 ((isACLiC) ? ModuleName : std::string("module")) + ".modulemap");
 
       // FIXME: We should only complain for modules which we know to exist. For example, we should not complain about
       // modules such as GenVector32 because it needs to fall back to GenVector.
-      ModuleWasSuccessfullyLoaded = LoadModule(ModuleName, *fInterpreter, /*Complain=*/ !isACLiC);
+      ModuleWasSuccessfullyLoaded = LoadModule(ModuleName, *fInterpreter, /*Complain=*/true);
```

Comment:
This was the only remaining use of `Complain == false`, please remove the parameter.